demo/src/features/debug3d/prepare.rs

```diff
@@ -1,51 +1,57 @@
-use super::write::Debug3dCommandWriter;
-use crate::features::debug3d::{
-    Debug3dDrawCall, Debug3dRenderFeature, Debug3dUniformBufferObject, Debug3dVertex,
-    ExtractedDebug3dData,
-};
+rafx::declare_render_feature_prepare_job!();
+
+use super::public::debug3d_resource::LineList3D;
+use super::write::Debug3DVertex;
+use super::RenderFeatureType;
 use crate::phases::OpaqueRenderPhase;
 use rafx::api::{RafxBufferDef, RafxMemoryUsage, RafxResourceType};
 use rafx::framework::{MaterialPassResource, ResourceArc};
-use rafx::nodes::{
-    FeatureCommandWriter, FeatureSubmitNodes, FramePacket, PrepareJob, RenderFeature,
-    RenderFeatureIndex, RenderJobPrepareContext, RenderView, ViewSubmitNodes,
-};
 
-pub struct Debug3dPrepareJobImpl {
+pub type Debug3dUniformBufferObject = shaders::debug_vert::PerFrameUboUniform;
+
+pub struct PrepareJobImpl {
```

Comment:
If we rename, would rename to Debug3dPrepareJob

Reply:
Resolving to consolidate discussion of job names / prefixes into https://github.com/aclysma/rafx/pull/122#discussion_r610704300.